### PR TITLE
add target to validate swagger with spectral

### PR DESCRIPTION
**Pull request by tupyy**
This PR address the issue #155. 

The rules set is the default one for OpenAPI.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -197,6 +197,9 @@ ifeq (, $(shell which ginkgo))
 	$(call go-get-tool,$(GINKGO),github.com/onsi/ginkgo/ginkgo@v1.16.5)
 endif
 
+validate-swagger: ## Validate swagger
+	$(DOCKER) run -it --rm -v $(PWD)/.spectral.yaml:/tmp/.spectral.yaml -v $(PWD)/swagger.yaml:/tmp/swagger.yaml stoplight/spectral lint --ruleset "/tmp/.spectral.yaml" /tmp/swagger.yaml
```
Comment (masayag) [inline Makefile:201]: pls remove trailing whitespace

Comment (masayag) [reply Makefile:201]: don't think it is needed here. the file should exist only inside a container that will be removed after the execution ends.

**Review by masayag — APPROVED** [review]

**Comment by masayag** [comment]
@tupyy Welcome and Thank you! :-)

**Review by gciavarrini — APPROVED** [review]
@tupyy Welcome :)

**Comment by tupyy** [comment]
@masayag  @gciavarrini  Thank you! :)

**Pull request merged**